### PR TITLE
Reagent Amount poured into trays fix

### PR DESCRIPTION
**Pull request by Nitre88**
[Hydroponics] [QOL]
## About the PR 
Fixes issue https://github.com/goonstation/goonstation/issues/14804
Adds correct amount poured from watering cans(and other containers) when used on hydroponic trays

## Why's this needed? 
The game currently shows the amount per transfer which is the maximum instead of what the reagents are actually transfering.
This would improve the correctness of ingame feedback your given. When interacting with hydroponics

### CODE REVIEW

**Comment by Nitre88** [comment]
Accidentally committing my last pr to my master master branch turned the 10 mins of coding and testing, into 2 hours of pain

File: code/obj/machinery/plantpot.dm [hunk 1]
```diff
@@ -701,7 +701,8 @@ TYPEINFO(/obj/machinery/plantpot)
 				boutput(user, "<span class='alert'>There is nothing in [W] to pour!</span>")
 				return
 			else
-				user.visible_message("<span class='notice'>[user] pours [W:amount_per_transfer_from_this] units of [W]'s contents into [src].</span>")
+				//corrects the amount of reagents shown to have been used when pouring into a tray
+				user.visible_message("<span class='notice'>[user] pours [min(W:amount_per_transfer_from_this,W.reagents.total_volume)] units of [W]'s contents into [src].</span>")
```
Comment (Tarmunora) [inline code/obj/machinery/plantpot.dm:705]: why not just use the return value from `trans_to()`?

Comment (Nitre88) [reply code/obj/machinery/plantpot.dm:705]: I was looking at how chem/beakers did it and used that
[https://github.com/goonstation/goonstation/blob/262b3faab2014af32ed03bed6df6bcc916328722/code/modules/chemistry/Chemistry-Tools.dm#L201C1-L201C1](url)

Comment (Tarmunora) [reply code/obj/machinery/plantpot.dm:705]: because that code block isn't for transferring, it's for splashing the beaker on something

Comment (Nitre88) [reply code/obj/machinery/plantpot.dm]: thanks

**Pull request merged**